internal/lsp/range.go

```diff
@@ -16,6 +16,21 @@ func HCLRangeToLSP(hclRng hcl.Range) lsp.Range {
 			Line:      hclRng.End.Line - 1,
 		},
 	}
+
+	if r.Start.Character < 0 {
+		r.Start.Character = 0
+	}
+	if r.Start.Line < 0 {
+		r.Start.Line = 0
+	}
+	if r.End.Character < 0 {
+		r.End.Character = 0
+	}
+	if r.End.Line < 0 {
+		r.End.Line = 0
+	}
```

Comment:
Was this added as a reaction to any particular edge case you ran into?

Reply:
HCL range start with index 1, in the event there is no subject, I passed a zero'd HCL range to the converter, this would result in negative values in the LSP conversion (since LSP range is index 0), this just ensures we don't send an invalid range